### DISCUSSION TITLE
Multiline TextEdit that fills entire CentralPanel

### DISCUSSION THREAD

**Discussion by elijah-potter** [emilk/egui#616] [Q&A] [answered]
Sorry if this is an obvious thing I am unaware of (I am very new to this library). 

I want to fill the entire CentralPanel with a multi line TextEdit. Is this possible?

**Comment by emilk** [comment, marked as answer]
Yes:

``` rust
ui.add_sized(ui.available_size(), TextEdit::multiline(&mut self.text));
```
